thesis-gui/src/components/auth/sign-up/SignUp.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import {
+    IonButton,
+    IonContent, IonHeader, IonInput,
+    IonPage, IonRouterLink, IonTitle, IonToolbar,
+} from "@ionic/react";
+import './SignUp.css';
+
+const SignUp: React.FC = () => {
+    return (
+        <IonPage>
+            <IonHeader>
+                <IonToolbar>
+                    <IonTitle>Sign Up</IonTitle>
+                </IonToolbar>
+            </IonHeader>
+            <IonContent className="ion-padding">
+                <IonHeader collapse="condense">
+                    <IonToolbar>
+                        <IonTitle size="large">Sign Up</IonTitle>
+                    </IonToolbar>
+                </IonHeader>
+                <IonInput type="text" required placeholder="Nickname" />
```

Comment:
What do you think about 
```
<IonItem>
       <IonLabel position="floating">Login</IonLabel>
       <IonInput type="text" required/>
</IonItem>
```
?
I think it looks a bit better.

The same applies to other fields.